src/main/resources/langs/lang-zh_CN.yml

```diff
@@ -150,135 +150,130 @@ Buttons:
+    Description: "&7将价格设置为零"
   CircleSetItem:
     Name: " "
   CircleUnsetItem:
     Name: " "
   YesButton:
-    Name: "确定"
+    Name: "是"
   NoButton:
-    Name: "取消"
+    Name: "否"
   Question:
     Name: " "
   SearchItemDrop:
-    Name: "&f查找类似物品"
-    Description: "&7使用 SHIFT+左键;&7点击你想要的物品"
+    Name: "&f寻找相似物品"
+    Description: "&7将待搜寻物品放入此处;&7或使用 Shift+鼠标左键"
   SearchItemText:
-    Name: "&f根据物品名字查找"
+    Name: "&f按名称寻找拍卖"
   SearchByPlayer:
-    Name: "&f根据玩家名字查找"
+    Name: "&f按玩家寻找拍卖"
   LastAuctions:
-    Name: "&3最近 {hours} 小时的拍卖"
+    Name: "&3{hours} 小时内发起的拍卖"
   LastCreatedAuctions:
-    Name: "&3 近{hours} 小时内创建的拍卖"
+    Name: "&3{hours} 小时内发起的拍卖"
   LastBoughtAuctions:
-    Name: "&3 近{hours} 小时内购买的拍卖品"
+    Name: "&3{hours} 小时内发起的拍卖"
   AdvicePriceInfo:
-    Name: "建议价格"
-    Description: "&3{advicePrice};&7点击使用建议价格"
+    Name: "建议售价"
+    Description: "&3{advicePrice};&7点击将价格设置为建议售价"
   SeeShulkerBoxContent:
-    Name: "查看该潜影盒的内容"
+    Name: "查看潜影盒内容"
 ErrorMessages:
-  UnknownWorld: "未知商店"
-  ShopTypeNotSet: "未设置商店类型；&2请至少选择一种类型"
-  MissingItems: "物品数量错误"
+  UnknownWorld: "商店所在世界未知"
+  ShopTypeNotSet: "商店类型未设置;&2请选择至少一种"
+  MissingItems: "物品数量不正确, 部分物品缺失"
   FailCreateAuction: "在数据库中创建拍卖失败"
-  NotEnoughSpace: "你的背包满了，请清理背包后重试"
-  NotEnoughMoney: "你没有足够的钱购买"
-  CantBuyAuction: "未能购买拍卖品（过期或已售出)"
-  CantRenewAuction: "拍卖续约失败"
-  CantUndoAuction: "未能撤消拍卖"
-  CantRenewEveryAuction: "延长时间错误"
-  CantRemoveAuction: "未能成功删除该拍卖"
-  CantUndoEveryAuction: "拍卖失败"
-  PlayerDoesntExist: "这个玩家不存在"
-  MaxAuctionByPlayer: "无法创建更多拍卖；&7您已达到最大拍卖数量"
-  PermissionNotAllowed: "&c不允许执行此操作"
-  InvalidCommand: "&c此命令无效 : "
-  PlayerOnly: "&c这个指令只能玩家使用"
-  UnknownShop: "&c未知商店 : "
-  TooManyShopMatching: "&cThere is too many shop matching {shopName}. Please specify the location"
-  CantRemoveBlock: "&c您不能删除此块，因为它附加了一个商店"
-  DamagedItem: "你不能卖损坏的东西"
-  BlacklistedItem: "你不能卖黑名单的商品"
-  CommandInChat: "&c不允许在此模式下执行命令"
-  MaxGlobalShop: "&c你已经达到创建商店的最大值了 ({maxGlobalShop} GlobalShop)"
-  WorldNotAllowed: "&cYou can't create a shop in this world"
+  NotEnoughSpace: "添加物品失败, 玩家背包空间不足"
+  NotEnoughMoney: "你没有足够的钱参与此次拍卖"
+  CantBuyAuction: "购买物品失败 (拍卖物品已过期或已售出)"
+  CantRenewAuction: "刷新拍卖失败"
+  CantUndoAuction: "删除拍卖失败"
+  CantRemoveAuction: "移除拍卖失败"
+  CantRenewEveryAuction: "刷新所有拍卖失败"
+  CantUndoEveryAuction: "删除所有拍卖失败"
+  PlayerDoesntExist: "该玩家不存在"
+  MaxAuctionByPlayer: "你不能再创建更多拍卖了;&7你已达到可发起拍卖场次的数量上限"
+  PermissionNotAllowed: "&c你不能这么做"
+  InvalidCommand: "&c命令无效: "
+  PlayerOnly: "&c该命令只能被玩家执行"
+  UnknownShop: "&c未知商店: "
+  CantRemoveBlock: "&c你不能破坏绑定了商店的方块"
+  DamagedItem: "你不能出售带耐久的物品"
+  BlacklistedItem: "你不能出售黑名单物品"
+  CommandInChat: "&c该状态下不能执行命令"
+  MaxGlobalShop: "&c你不能再创建更多全球商店了 (最多 {maxGlobalShop} 个)"
 InfoMessages:
-  UndoAuction: "&2删除成功"
-  RemoveAuction: "&2成功删除该拍卖"
-  RenewAuction: "&2物品已经成功续期"
-  CreateAuction: "&2创建成功"
-  RenewEveryAuction: "&2所有物品已经成功续期"
-  UndoEveryAuction: "&2所有物品已经成功删除"
-  ShopCreated: "Shop successfully created"
```

Comment:
Can you translate this sentence instead of deleting it ?

Reply:
```  ShopCreated: "商店创建成功"```